Tasks/app-store-release/app-store-release.ts

```diff
@@ -33,7 +33,14 @@ function isValidFilePath(filePath: string): boolean {
 // Attempts to find a single ipa file to use by the task.
 // If a glob pattern is provided, only a single ipa is allowed.
 function findIpa(ipaPath: string): string {
-    let paths: string[] = tl.findMatch('', ipaPath);
+    // We need to allow broken symlinks since there could be broken symlinks found in source folder, but filtered by contents pattern
```

Comment:
This looks not relevant here - there's no Contents input for app-store-release task

Reply:
yes, _ipaPath pattern_ will sound better, I assume?